<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateShopItemWidget</class>
 <widget class="QWidget" name="CreateShopItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create new item</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dino</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Engram</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Beacon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Experience</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Command</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="submitBtn">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBtn">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
